feature/COM-2886

prakhar308 opened the pull request:
- Added link in Header to open B2B Account Request form
 - Added validations
 - Added API Integration
 - Displayed success popup message

AtulChavan commented:
@Rameshdesaiise I have fixed build issue.
As suggested by @michaelsadlonkibo I have also removed tenant attribute check and made B2B link available to all.

rakesh-dontula reviewed (changes requested):
change b2b request account to b2b account request everywhere

Edit : Looks like the change is already made. looks good.

prakhar308 committed:
COM-2886: Created feature for B2B Account Request

- Added link in Header to open B2B Account Request form
- Added validations
- Added API Integration
- Displayed success popup message

prakhar308 committed:
COM-2886: Use Mozu email validation pattern and change color of asterisk
